api/AnimationEffect.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEffect",
         "support": {
           "chrome": {
-            "version_added": null
+            "version_added": true
```

Comment:
This is the first change remaining on top of my 4 PRs. This is disconfirmed in DevTools, the `AnimationEffect` isn't exposed. That's because it's `RuntimeEnabled=WebAnimationsAPI`:
https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/animation/animation_effect.idl?l=34&rcl=d05a9eee63f8455e1333c4fde0ea37c44c97a00c

And that's not enabled by default:
https://cs.chromium.org/chromium/src/third_party/blink/renderer/platform/runtime_enabled_features.json5?l=1480&rcl=d05a9eee63f8455e1333c4fde0ea37c44c97a00c

Without considering the various extended attributes in Chromium that can change whether an API is exposed, there will be many false positives.

Reply:
There seems to be a lot of issues regarding the runtime flags during detection.  I dismissed them as I knew I'd be going through manually when looking up the version numbers (though I realize it was a little dumb).